R/dataframe__frame.R

```diff
@@ -1,34 +1,52 @@
-#' and private methods/function calls to the polars Rust side. The instantiated
-#' `DataFrame`-object is an `externalptr` to a low-level Rust polars DataFrame
-#' object.
-#'
-#' The S3 method `.DollarNames.RPolarsDataFrame` exposes all public `$foobar()`-methods
-#' which are callable onto the object. Most methods return another `DataFrame`-
-#' class instance or similar which allows for method chaining. This class system
-#' could be called "environment classes" (in lack of a better name) and is the
-#' same class system `extendr` provides, except here there are both a public and
-#' private set of methods. For implementation reasons, the private methods are
-#' external and must be called from `.pr$DataFrame$methodname()`. Also, all
-#' private methods must take any `self` as an argument, thus they are pure
-#' functions. Having the private methods as pure functions solved/simplified
-#' self-referential complications.
-#'
-#' @details
-#' Check out the source code in [R/dataframe_frame.R](https://github.com/pola-rs/r-polars/blob/main/R/dataframe__frame.R)
+#' @description The `DataFrame`-class is simply two environments of respectively
+#' the public and private methods/function calls to the polars Rust side. The
+#' instantiated `DataFrame`-object is an `externalptr` to a low-level Rust
+#' polars DataFrame object.
+#'
+#' The S3 method `.DollarNames.RPolarsDataFrame` exposes all public
+#' `$foobar()`-methods which are callable onto the object. Most methods return
+#' another `DataFrame`- class instance or similar which allows for method
+#' chaining. This class system could be called "environment classes" (in lack
+#' of a better name) and is the same class system `extendr` provides, except
+#' here there are both a public and private set of methods. For implementation
+#' reasons, the private methods are external and must be called from
+#' `.pr$DataFrame$methodname()`. Also, all private methods must take any
+#' `self` as an argument, thus they are pure functions. Having the private
+#' methods as pure functions solved/simplified self-referential complications.
+#'
+#' @section Active bindings:
+#'
+#' ## flags
+#'
+#' `$flags` returns a nested list with column names at the top level and
+#' column flags in each sublist.
+#'
+#' Flags are used internally to avoid doing unnecessary computations, such as
+#' sorting a variable that we know is already sorted. The number of flags
+#' varies depending on the column type: columns of type `array` and `list`
+#' have the flags `SORTED_ASC`, `SORTED_DESC`, and `FAST_EXPLODE`, while other
+#' column types only have the former two.
+#'
+#' - `SORTED_ASC` is set to `TRUE` when we sort a column in increasing order, so
+#'   that we can use this information later on to avoid re-sorting it.
+#' - `SORTED_DESC` is similar but applies to sort in decreasing order.
+#'
+#' @details Check out the source code in
+#' [R/dataframe_frame.R](https://github.com/pola-rs/r-polars/blob/main/R/dataframe__frame.R)
```

Comment:
I don't think such things should be placed here (they should be in the developer's documentation), but since they are irrelevant to this PR, I'll leave them alone.